DOL.WHD.Section14c.Web/src/modules/components/dateField/dateFieldTemplate.html

```diff
@@ -1,14 +1,20 @@
-<div class="usa-date-of-birth">
+<div class="usa-date-of-birth dol-side-example">
   <div class="usa-form-group usa-form-group-month">
     <label for="{{::$id}}_Month">Month</label>
-    <input class="usa-input-inline" aria-describedby="dobHint" class="usa-form-control" id="{{::$id}}_Month" name="{{::$id}}_Month" pattern="0?[1-9]|1[012]" type="number" min="1" max="12" value="" ng-model="vm.month">
+    <input class="usa-input-inline" aria-describedby="{{::$id}}_dateExample {{::$id}}_dateExampleMonth" class="usa-form-control" id="{{::$id}}_Month" name="{{::$id}}_Month" pattern="0?[1-9]|1[012]" type="number" min="1" max="12" value="" ng-model="vm.month">
```

Comment:
Yay, :heart: one-time bindings!  Also, today I learned you can put multiple IDs in an `aria-describedby` - cool!